compiler/Eta/Rename/RnSplice.hs

```diff
@@ -831,7 +831,7 @@ A thing can have a bind_lvl of outerLevel, but have an internal name:
    foo = [d| op = 3
              bop = op + 1 |]
 Here the bind_lvl of 'op' is (bogusly) outerLevel, even though it is
-bound inside a bracket.  That is because we don't even even record
+bound inside a bracket.  That is because we don't even record
```

Comment:
?

Sometimes words are doubled for emphasis. I don't think this is a case, but...